frame/instrumental-strategy-pablo/src/tests.rs

```diff
@@ -12,7 +20,7 @@ use crate::{pallet, pallet::Error};
 // -------------------------------------------------------------------------------------------------
 
 #[test]
-fn test_add_an_associated_vault() {
+fn add_an_associated_vault() {
```

Comment:
I don't know about naming of test standards. There are many articles about it ([discussion in Slack](https://composablefinance.slack.com/archives/C02QK64F928/p1654595338307759) for example), but there is no official answer.

> Why did you remove test_..?

Because:
- we already have `tests` in the filename;
- we already have `#[cfg(test)]` attribute for the module;
- we already have `#[test]` attribute for functions;
- when we run `cargo test`, we only get information about tests function, so it doesn't make sense to duplicate this information (`test_*`) in the name of a function.

Reply:
Got it. Thanks for detailed response!